zitadel/provider.go

```diff
@@ -49,6 +49,7 @@ import (
 	"github.com/zitadel/terraform-provider-zitadel/v2/zitadel/idp_google"
 	"github.com/zitadel/terraform-provider-zitadel/v2/zitadel/idp_ldap"
 	"github.com/zitadel/terraform-provider-zitadel/v2/zitadel/idp_oauth"
+	"github.com/zitadel/terraform-provider-zitadel/v2/zitadel/idp_oidc"
```

Comment:
Could you please also add the "zitadel_idp_oidc" resource and datasource?

Reply:
Updated @stebenz

Reply:
Perfect, thank you